src/translations/components/Map.ts

```diff
@@ -4,15 +4,15 @@ const MapTexts = {
   exitButton: {
     a11yLabel: _('Gå tilbake', 'Go back'),
   },
-  expandButton: {
-    label: _('Utvid kart', 'Expand map'),
-  },
   startPoint: {
     label: _('Start', 'Start'),
   },
   endPoint: {
     label: _('Slutt', 'End'),
   },
+  showTrip: {
+    label: _('Vis reiserute i kart', 'Show trip in map'),
+  },
```

Comment:
Should the text be in `MapTexts` now, when it is passed in from outside the map?